src/code/Microsoft.PowerShell.ThreadJob.cs

```diff
@@ -501,6 +511,22 @@ public JobDefinition ThreadJobDefinition
         static ThreadJob()
         {
             s_JobQueue = new ThreadJobQueue(5);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Assembly assembly = typeof(PSObject).Assembly;
+                Type systemPolicy = assembly.GetType("System.Management.Automation.Security.SystemPolicy");
+                s_getSystemLockdownPolicy = systemPolicy.GetMethod("GetSystemLockdownPolicy", BindingFlags.Public | BindingFlags.Static);
+                Type systemEnforcementMode = assembly.GetType("System.Management.Automation.Security.SystemEnforcementMode");
+                s_enforce = systemEnforcementMode.GetField("Enforce");
+                s_enforceValue = s_enforce.GetValue(null);
+                s_getFileLockdownPolicy = systemPolicy.GetMethod("GetLockdownPolicy", BindingFlags.Public | BindingFlags.Static, null, new Type[] { typeof(string), typeof(SafeHandle) }, null);
+            }
```

Comment:
Add a new line to make it more readable.
```suggestion
            }

```

Reply:
done